enthought_example/example_data_views/example_data_view.py

```diff
@@ -0,0 +1,31 @@
+from force_wfmanager.ui.review.plot import BasePlot, ChacoPlot
+
+
+class ExampleCustomPlot(BasePlot):
+    """ This is an example of a custom plot, that replaces
+    the default scatter plot with a line plot.
+
+    """
+
+    title = 'Line plot'
+
+    description = 'Example line plot'
+
+    def plot_line(self):
+        plot = ChacoPlot(self._plot_data)
```

Comment:
Could limit this to something like 
```suggestion
        n_max = 2000
        plot = ChacoPlot(self._plot_data[:n_max])
```

Reply:
This is strange anyway. If I launch a wfmanager, select line plot, then load an existing analysis with 3000 points, it will display it fine. I'll add the workaround for now, but this may need investigation.

Reply:
Addressing it in `wfmanager`. Thanks for pointing this out.

Reply:
This issue (which isn't new: it's just made evident by the line plot) is larger in scope. See https://github.com/force-h2020/force-wfmanager/issues/320